tests/topotests/bgp_nhc/r1/frr.conf

```diff
@@ -8,6 +8,7 @@ int r1-eth1
 router bgp 65001
  bgp router-id 10.255.0.1
  no bgp ebgp-requires-policy
+ bgp bestpath as-path multipath-relax
```

Comment:
This change by itself seems odd.  Do we need further test checking of the route being installed into zebra? and that the weights are what we expect them to be?